src/python/bot/minimizer/minimizer.py

```diff
@@ -208,18 +208,21 @@ def _delete_file_if_needed(self, input_file):
     except OSError:
       pass
 
-  def _report_progress(self):
+  def _report_progress(self, is_final=False):
     """Call a function to report progress if the minimizer uses one."""
-    if not self.minimizer.progress_report_function:
+    if not self.minimizer.progress_report_function and not is_final:
```

Comment:
In this case, we still want to bail out. I believe this would lead to an exception below since we wouldn't have a valid progress report function to call.